common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +50,34 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    def test_logger_backend_unicode_character(self):
+        """
+        When event information contain utf and latin1 characters
+        """
+        # Utf-8 characters
+        event_string_unicode = {'string': '测试'}
+        event_unicode_characters = {'string': u'测试'}
+        event_encoded_unicode_characters_string = {
+            'string': event_unicode_characters['string'].encode('utf-8')
+        }  # pylint: disable=invalid-name
+        # Latin1 characters
+        event_string_latin_characters = {'string': 'Ó é ñ'}
+        event_unicode_latin_characters = {'string': u'Ó é ñ'}
+        event_encoded_latin_characters_string = {
+            'string': event_unicode_latin_characters['string'].encode('latin1')
+        }  # pylint: disable=invalid-name
+
+        self.backend.send(event_string_unicode)
+        self.backend.send(event_unicode_characters)
+        self.backend.send(event_encoded_unicode_characters_string)
```

Comment:
Why don't these fail?   If this is encoded as utf-8, then decoding as latin1 shouldn't work.  And you're not checking the output anyway.